docs/EDGEVIEW-CONTAINER-API.md

```diff
@@ -23,3 +23,69 @@ For security reasons, the Edge-View container has all the volumes mounted in 're
 ## Logging to Event
 
 Edge-View logging is similar to other containers. For any user command received by Edge-View, it will log the client endpoint(IP address/port), the command and its parameters. The log entry will also be tagged with object-type of `log-to-event`, and the controller can optionally process those log entries and generate them as device events or alerts.
+
+## Edge-View Client Authentication
+
+To address limitations in controlling and tracking user access to Edge-View commands, the authentication mechanism to Edge-View aim to strengthen user authentication, enforce stricter access controls, and improve user tracking.
```

Comment:
"user tracking" (here and below in the text) sounds a bit scary. Isn't "user auditing" a more common term?

Reply:
Ok, makes sense.